internal/manager/config_validation_test.go

```diff
@@ -244,7 +263,7 @@ func TestConfigValidate(t *testing.T) {
 
 		t.Run("enabled with no runtime group is rejected", func(t *testing.T) {
 			c := validEnabled()
-			c.Konnect.RuntimeGroupID = ""
+			c.Konnect.ControlPlaneID = ""
 			require.ErrorContains(t, c.Validate(), "runtime group not specified")
```

Comment:
Could we also update the error message here or would that be too much to do in this PR?